monash-human-power/dashboard
# Refactor chart rendering into generic component

## Description

Split the data fetching and chart rendering parts into separate components to prepare for V3 views.

## Screenshots

N/A

## Steps to Test

Open dashboard and start the mock data test script. Everything should still work as before.
